shared/common-adapters/avatar.native.js

```diff
@@ -23,13 +24,25 @@ export default class Avatar extends Component<void, Props, State> {
   render () {
     const {size} = this.props
     const uri = {uri: shared.createAvatarUrl(this.props)}
+    const propsOpacity = this.props.hasOwnProperty('opacity') ? this.props.opacity : 1.0
+    const opacity = this.state.avatarLoaded ? propsOpacity : 0
 
     return (
-      <TouchableOpacity style={{...stylesContainer(size), ...this.props.style}} disabled={!this.props.onClick} onPress={this.props.onClick} activeOpacity={0.8}>
+      <TouchableOpacity
+        style={{...stylesContainer(size), ...this.props.style}}
+        disabled={!this.props.onClick}
+        onPress={this.props.onClick}
+        activeOpacity={0.8}>
         <Box style={stylesContainer(size)}>
+          {this.props.hasBackgroundColor &&
+            <View
+              style={_.omit({...stylesImage(size),
+                resizeMode: undefined,
+                backgroundColor: this.props.hasBackgroundColor,
+              }, 'resizeMode')} />}
```

Comment:
Why call `omit` with `'resizeMode'` and also set `resizeMode: undefined` in the object?